API/common/utils.py

```diff
@@ -313,6 +313,16 @@ def last_commit_info(git_repo_path):
     return info
 
 
+def parse_as_int(value):
```

Comment:
This does not do any parsing, just returns the value if it is string otherwise 0.